pkg/deprecatetemplates/allowlist.go

```diff
@@ -0,0 +1,90 @@
+package deprecatetemplates
+
+import (
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+type deprecatedTemplateBlocker struct {
+	Jira        string      `json:"jira"`
+	Description string      `json:"description"`
+	Jobs        sets.String `json:"jobs"`
+}
+
+type deprecatedTemplate struct {
+	Name           string                               `json:"template_name"`
```

Comment:
If this is stored in a map keyed by name, why do we need this field?

Reply:
I wanted the structure to be independent, carrying all relevant informaiton. Carry the pointer to the JIRA so it is usable even without passing the key separately. 

But I guees I could do that programatically on load without polluting the file, and only add that when it's useful. :+1: